src/components/EtGrid.vue

```diff
@@ -125,4 +123,10 @@ export default {
 
 .q-hide-add
   display none
+
+.input-search
+  width 300px
+
+.icon
```

Comment:
.icon -> .et-icon
need to prefix newly created class with .et-... to avoid conflict